src/views/SinglePostView.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+import axios from 'axios'
+import LoaderSpinner from '../components/LoaderSpinner.vue'
+import PostComment from '../components/PostComment.vue'
+import type { Post, Comment } from '../types/interfaces'
+
+const route = useRoute()
+const post = ref<Post | null>(null)
+const comments = ref<Comment[]>([])
+const isLoading = ref(true)
+
+onMounted(async () => {
+  const postId = route.params.id
+  try {
+    const postResponse = await axios.get(`https://jsonplaceholder.typicode.com/posts/${postId}`)
```

Comment:
We would have liked to see a global variable with the host, or the api_base_url, or something like this instead of repeating `https://jsonplaceholder.typicode.com/`

Reply:
Done on api.ts file